common.mk

```diff
@@ -65,29 +65,30 @@ HELP_COMMANDS += \
 # include additional subproject make fragments
 # see HELP_COMPILATION_VARIABLES
 #########################################################################################
-include $(base_dir)/generators/cva6/cva6.mk
-include $(base_dir)/generators/ibex/ibex.mk
-include $(base_dir)/generators/ara/ara.mk
 include $(base_dir)/generators/tracegen/tracegen.mk
-include $(base_dir)/generators/nvdla/nvdla.mk
-include $(base_dir)/generators/radiance/radiance.mk
 include $(base_dir)/tools/torture.mk
+-include $(base_dir)/generators/cva6/cva6.mk
+-include $(base_dir)/generators/ibex/ibex.mk
+-include $(base_dir)/generators/ara/ara.mk
+-include $(base_dir)/generators/nvdla/nvdla.mk
+-include $(base_dir)/generators/radiance/radiance.mk
+
 
 #########################################################################################
 # Prerequisite lists
 #########################################################################################
 # Returns a list of files in directories $1 with single file extension $2.
 # If available, use 'fd' to find the list of files, which is faster than 'find'.
 ifeq ($(shell which fd 2> /dev/null),)
-	lookup_srcs = $(shell find -L $(1)/ -name target -prune -o \( -iname "*.$(2)" ! -iname ".*" \) -print 2> /dev/null)
+	lookup_srcs = $(shell find -L $(1)/ -name target -prune -o \( ! -xtype l -a -iname "*.$(2)" ! -iname ".*" \) -print 2> /dev/null)
```

Comment:
Can we make this find expression easier to read, i.e. `not a symlink and (name is $(2) and not name .*)`. Currently, this is quite hard to parse.